Cargo.toml

```diff
@@ -52,5 +52,5 @@ members = [
 [features]
 # Enable all supported architectures by default.
 default = ["cranelift-codegen/all-arch"]
-lightbeam = ["wasmtime-environ/lightbeam", "wasmtime-jit/lightbeam"]
+lightbeam = ["wasmtime-environ/lightbeam", "wasmtime-jit/lightbeam", "wasmtime-wast/lightbeam"]
```

Comment:
Is the goal to remove lightbeam as a feature in future patches?

Note I tried:
cargo build
target/debug/wasm2obj --always-lightbeam  ./fib2.wasm -o fib2.lightbeam.o
target/debug/wasm2obj --always-cranelift  ./fib2.wasm -o fib2.cranelift.o

cargo build --features lightbeam
target/debug/wasm2obj --always-lightbeam  ./fib2.wasm -o fib2.lightbeam.o
target/debug/wasm2obj --always-cranelift  ./fib2.wasm -o fib2.cranelift.o

And all for objdumps produced the same cranelift based code. I can't see how to produce lightbeam based objdump. Is this expected?

Reply:
I expect we'll keep lightbeam behind a cargo feature for now, because it currently depends on dynasm which only works with Rust nightly builds. If that changes, we can re-evaluate.

wasm2obj --always-lightbeam should enable lightbeam though. I'll take a look at that.